frontend/src/components/G6Graph.svelte

```diff
@@ -1,27 +1,27 @@
 <script lang="ts">
-    import { onMount } from "svelte";
```

Comment:
the auto format is a bit unfortunate, I know our config is not enforced atm but could you follow the style for now so that the diff is not ugly like this?

Reply:
Oh sorry, I did not think about this when formatting :x (stupid habits 😠)
I will notify the others about that!